envoy/go.mod

```diff
@@ -1,33 +1,39 @@
 module github.com/cybozu/neco-containers/envoy
 
-go 1.20
+go 1.21
+
+toolchain go1.21.4
```

Comment:
Is this directive required?

Reply:
It wouldn't be required. I think it was added somehow when I update its dependencies. I'll remove it :)